src/Microsoft.AspNet.WebHooks.Custom.SqlStorage/Extensions/HttpConfigurationExtensions.cs

```diff
@@ -35,7 +35,10 @@ public static void InitializeCustomWebHooksSqlStorage(this HttpConfiguration con
         /// </summary>
         /// <param name="config">The current <see cref="HttpConfiguration"/>config.</param>
         /// <param name="encryptData">Indicates whether the data should be encrypted using <see cref="IDataProtector"/> while persisted.</param>
-        public static void InitializeCustomWebHooksSqlStorage(this HttpConfiguration config, bool encryptData)
+        /// <param name="connectionStringName">Sets custom name of connection string <see cref="WebHookStoreContext"/></param>
+        /// <param name="schemaName">Sets custom name of schema <see cref="WebHookStoreContext"/></param>
+        /// <param name="tableName">Sets custom name of table <see cref="WebHookStoreContext"/></param>
```

Comment:
- The `connectionStringName` parameter is passed directly to the `DbContext` constructor. Therefore, it's actually a `nameOrConnectionString`. Change the parameter and the `WebHookStoreContext` property's names to `nameOrConnectionString` / `NameOrConnectionString`.
- end all descriptions with a period
- `<see cref="..."/>` just adds a link, doesn't add words like "see also"
- better to reference the specific `WebHookStoreContext` properties in these descriptions (if all remain) or add a general `<see cref="WebHookStoreContext"/>` below these `<param .../>` elements.

E.g.
``` c#
<param name="nameOrConnectionString">The custom connection string or name of the connection string application setting. Used to initialize <see cref="WebHookStoreContext.ConnectionStringName"/>.</param>
```

Reply:
One question, should be `NameOrConnectionString` or `NameOfConnectionString` ?